lib/matplotlib/lines.py

```diff
@@ -143,6 +143,9 @@ def _slice_or_none(in_v, slc):
     # if just an int, assume starting at 0 and make a tuple
     elif isinstance(markevery, int):
         markevery = (0, markevery)
+    # if just an numpy int, assume starting at 0 and make a tuple
```

Comment:
Wont this cause an issue for actual integers?
Because 4.items() would cause an error